src/test/java/stepDefinition/LoginStepDefinitions.java

```diff
@@ -0,0 +1,65 @@
+package stepDefinition;
+
+import PagesIvaF.HomePageIvan;
+import PagesIvaF.LoginPageIvan;
+import io.cucumber.java.After;
+import io.cucumber.java.Before;
+import io.cucumber.java.en.And;
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.Assert;
+
+import java.beans.Visibility;
+import java.time.Duration;
+
+public class LoginStepDefinitions {
+    WebDriver driver;
+    WebDriverWait wait;
+
+    @Before
+    public void openBrowser(){
+        driver = new ChromeDriver();
+        wait = new WebDriverWait(driver, Duration.ofSeconds(10));
+    }
+
+
+    @Given("I open Login Page")
+    public void iOpenLoginPage() {
+        driver.get("https://bbb.testpro.io/");
+    }
+
+    @When("I enter email {string}")
+    public void iEnterEmail(String arg0) {
+        LoginPageIvan loginPageIvan = new LoginPageIvan(driver);
+        loginPageIvan.provideEmail(arg0);
+
+    }
+
+    @And("I enter password {string}")
+    public void iEnterPassword(String arg0) {
+        LoginPageIvan loginPageIvan = new LoginPageIvan(driver);
+        loginPageIvan.providePassword(arg0);
+
+    }
+
+    @And("I submit")
+    public void iSubmit() {
+        LoginPageIvan loginPageIvan = new LoginPageIvan(driver);
+        loginPageIvan.clickSubmitBtn();
+    }
+
+    @Then("I am logged in")
+    public void iAmLoggedIn() {
+        LoginPageIvan loginPageIvan = new LoginPageIvan(driver);
+        HomePageIvan homePageIvan = new HomePageIvan(driver);
+        Assert.assertTrue(homePageIvan.isUserAvatarDisplayed());
```

Comment:
![image](https://user-images.githubusercontent.com/104761482/202395007-7cd52841-e09d-4ffe-908a-c0abc00c1e89.png)

Disable this message with one of the following:

- src/test/resources/cucumber.properties: cucumber.publish.quiet=true
- src/test/resources/junit-platform.properties: cucumber.publish.quiet=true

![image](https://user-images.githubusercontent.com/104761482/202369035-8883092b-5407-43fc-8b4a-fc85ee5c5b44.png)

![image](https://user-images.githubusercontent.com/104761482/202396157-64764231-2f6d-4f7c-8213-3e750d8e078f.png)

![image](https://user-images.githubusercontent.com/104761482/202396211-c7b3e6c2-888d-488b-b377-cdeee446e516.png)


Add `publish = true`

```
@CucumberOptions
    (features = {"src/test/resources/features/Login.feature"},publish = true)
```
![image](https://user-images.githubusercontent.com/104761482/202396461-5f30eb6e-cae0-40af-82a6-d1fbff56cb62.png)

Result:

![image](https://user-images.githubusercontent.com/104761482/202396587-2ffe0a0e-e91f-49be-a779-cc641fafb0af.png)